src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodTypeParamsTags.java

```diff
@@ -61,9 +61,9 @@ public <L> void doSomethingElse2(L aAnEl) // ok
 
     /**
      * Some explanation.
-     * @param <Z The wrong type param // violation
+     * @param <Z The wrong type param // violation 'Unused @param tag for '<Z.'
```

Comment:
Please move out of javadoc , all violations in javadoc.
Please use `// violation x lines above|below...`

Reply:
Okay, I will make these changes. 

Is there a way I can run these tests locally before submitting a PR?

Reply:
>Is there a way I can run these tests locally before submitting a PR?

@sd9616 please read https://checkstyle.org/contributing.html#Submitting_your_contribution, make sure that `mvn clean verify` passes on your local before pushing changes to PR